frontend/src/lib/components/PreAggregatedBadge/PreAggregatedBadge.tsx

```diff
@@ -0,0 +1,13 @@
+import { IconBolt } from '@posthog/icons'
```

Comment:
I wanted to keep it inside web analytics, but then it felt wrong to import it into the DataTable, and wrapping a custom DataTable for our usage is too much.